src/Components/Event.php

```diff
@@ -13,17 +13,19 @@ class Event extends Component
     /**
      * Create a new component instance.
      *
-     * @return void
+     * @param mixed $listen
      */
-    public function __construct(public string $listen, public string $scope = '{ subscribed, events }')
+    public function __construct(public $listen, public string $scope = '{ subscribed, events }')
```

Comment:
In my opinionthe function signature should something like this:
`public function __construct(public mixed $listen, public string $scope = '{ subscribed, events }')`

Reply:
Thank you for your comment!

I think `mixed` needs higher version of PHP.
https://github.com/protonemedia/laravel-splade/blob/2f24f7d2140f15393b8d001c4262ccf66efd596e/composer.json#L19

Reply:
`mixed` has been included in PHP 8 ([rfc](https://wiki.php.net/rfc/mixed_type_v2)).

Reply:
You are right, Alex.